inspirehep/modules/workflows/workflows/article.py

```diff
@@ -180,12 +181,20 @@
         is_arxiv_paper,
         [
             arxiv_fulltext_download,
+            arxiv_package_download,
```

Comment:
Since we need to kill all arxiv harvesting anyway and restarting it it's no problem, as long as there are instead no user submissions stuck.